appcli/orchestrators.py

```diff
@@ -598,6 +599,239 @@ def __exec_command(self, command: Iterable[str]) -> CompletedProcess:
+            self.release_name,
+            "-n",
+            self.release_name,
+        ]
+
+        # Run the command.
+        return self.__run_command(command)
+
+    def status(
+        self, cli_context: CliContext, service_name: tuple[str, ...] = None
+    ) -> CompletedProcess:
+        """
+        Get the status of the chart (through `helm status`).
+
+        Args:
+            cli_context (CliContext): The current CLI context.
+
+        Returns:
+            CompletedProcess: Result of the orchestrator command.
+        """
+        # Generate the command string.
+        command = [
+            "helm",
+            "status",
+            self.release_name,
+            "-n",
+            self.release_name,
+        ]
+
+        # Run the command.
+        return self.__run_command(command)
+
+    def task(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        extra_args: Iterable[str],
+        detached: bool = False,
+    ) -> CompletedProcess:
+        logger.info("HelmOrchestrator has no services to run tasks for.")
+        return None
+
+    def exec(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        command: Iterable[str],
+        stdin_input: str = None,
+        capture_output: bool = False,
+    ) -> CompletedProcess:
+        logger.info("HelmOrchestrator does not support executing arbitrary commands.")
+        return None
+
+    def get_logs_command(self) -> click.Command:
+        @click.command()
+        def log():
+            logger.info("HelmOrchestrator does not support getting logs.")
+            return None
+
+        return log
+
+    def get_additional_commands(self):
+        # TODO: AF-248: Add `kubectl`, `helm` and possibly `k9s` as commands that can be called.
+        return []
+
+    def get_name(self) -> str:
+        return "helm"
+
+    def verify_service_names(
+        self, cli_context: CliContext, service_names: tuple[str, ...]
+    ) -> bool:
+        if service_names is None or len(service_names) == 0:
+            return True
+        logger.info("HelmOrchestrator has no services.")
+        return False
+
+    def get_disabled_commands(self) -> list[str]:
+        # The `task` command is disabled because helm is only used for install/upgrade/uninstall/downgrade operations.
+        # The `init` command is disabled because it's used to initialise additional services, and this orchestrator
+        # has no services. NOTE: The `init` command will be removed in the future.
+        return ["init", "task"]
+
+    def __run_command(self, command: list[str]) -> CompletedProcess:
+        """Run the given command and return the CompletedProcess.
+
+        Args:
+            command (list[str]): The command to execute.
+
+        Returns:
+            CompletedProcess: The execution result.
+        """
+        logger.debug(f"Executing {str(command)}")
+        result = subprocess.run(command, capture_output=True)
```

Comment:
We don't want to use `capture_output=False` so that it live prints progress? Isn't that what we do in the docker compose one?
Otherwise, isn't it just going to block for a long time while we have no idea what's going on?